lib/commands/general.js

```diff
@@ -176,6 +176,52 @@ commands.mobileBackdoor = async function (opts = {}) {
   return await this.espresso.jwproxy.command(`/appium/execute_mobile/backdoor`, 'POST', {target, methods});
 };
 
+
+/**
+ *  API to invoke methods defined on element views.
+ *
+ *  Example data
+ *{
+ *  "element": "1d737327-280d-4837-af5f-deacb0d002da",
+ *  "methods": [
+ *    {
+ *      "name": "someMethod"
+ *    },
+ *    {
+ *      "name": "anotherMethod",
+ *      "args": [
+ *        {
+ *          "value": "Lol",
+ *          "type": "String"
+ *        },
+ *        {
+ *          "value": true,
+ *          "type": "boolean"
+ *        }
+ *      ]
+ *    }
+ *  ]
+ *}
+ * In above example, method "someMethod" will be invoked on element with specified id. On the result, "anotherMethod" will be invoked
+ * - Only Public methods can be invoked
+ * - Arguments can also be passed
+ * - following primitive types are supported: "int", "boolean", "byte", "short", "long", "float", "char"
+ * -  Non-primitive types with fully qualified name "java.lang.*" is also supported:
+ *                              Eg. "java.lang.CharSequence", "java.lang.String", "java.lang.Integer", "java.lang.Float",
+ *                              "java.lang.Double", "java.lang.Boolean", "java.lang.Long", "java.lang.Short",
+ *                              "java.lang.Character" etc...
+ *
+ *
+ * @throws  {Error} if a method is not found with given argument types
+ *
+ * @return {*} the result of the last method in the invocation chain. If method return type is void, then "<VOID>" will be returned
+ *
+ */
+commands.mobileElementInvoke = async function (opts = {}) {
+  const {element, methods} = assertRequiredOptions(opts, ['element', 'methods']);
+  return await this.espresso.jwproxy.command(`/appium/execute_mobile/${util.unwrapElement(element)}/invoke_method`, 'POST', {methods});
```

Comment:
do we really need a separate endpoint for this? I think we could instead simply add one more target (aka element id) type into `mobile: backdoor` call

Reply:
Usually, I see elementId is a uri parameter, therefore tried to follow same practice and created a different endpoint. I can add another target "element" and additional parameter elementId in the POST